_extras/guide.md

```diff
@@ -137,6 +150,8 @@ This idiom violates the recommendation not to repeat variables names, but since
 
 * The Pan-STARRS join exercise is likely to feel scary to a lot of students - they have only seen one example. Emphasize the building blocks of a join e.g. FROM table1 JOIN table2 ON table1.common_column=table2.common_column. It is also worth pointing out that common_columns will not have the same names in this case
 
+* The `head` function is a python version of the Unix `head`. The Python version (which can be found in the episode_functions.py file) is used because it is platform independent so a single syntax can be used by all learners. You don't need to go through the function, but if someone asks, feel free to open the episode functions.py file and show them. 
```

Comment:
```suggestion
* The `head` function is a python version of the Unix `head` command. The Python version (which can be found in the episode_functions.py file) is used because it is platform independent so a single syntax can be used by all learners. You don't need to go through the function, but if someone asks, feel free to open the episode functions.py file and show them. 
```